### ISSUE TITLE
EXT: Typing `-` for key in KeyValuePair is reseting input

### ISSUE THREAD

**Issue by mrCherry97** [kyma-project/busola#2006]
**Description**
When user want to provide selector key in Create form like `test-123` input is removing `test-` and type only a `123`.

**Expected result**
User can provide `test-123` to key in KeyValuePair for example in selector.

**Actual result**
As in description.

**Steps to reproduce**
1. Go to Extensibility Gateways view
2. Click Create
3. Try to write `test-123` in Selector key input

**Troubleshooting**

**Comment by mrCherry97** [comment]
Now works fine.

**Comment by dariadomagala** [comment]
> Now works fine.

Nope. Still can reproduce - see Applications from examples and edit labels.
